inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/utils/ViewUtil.kt

```diff
@@ -33,4 +33,15 @@ internal object ViewUtil {
             null
         }
     }
+
+    fun getDisplayWidth(context: Context): Int {
+        val displayMetrics = context.resources.displayMetrics
+        return displayMetrics.widthPixels + 1
```

Comment:
Why add 1 pixel?

Reply:
The message view displayed a very small empty space, adding one pixel fixed the issue.

Reply:
Please add code comment